packages/terra-popup/src/terra-dev-site/doc/example/PopupAttachmentBehavior.jsx

```diff
@@ -1,35 +1,63 @@
-import React from 'react';
-import PopupCommonExample from 'terra-popup/lib/terra-dev-site/doc/common/PopupCommonExample';
+import React, { useRef, useState } from 'react';
+import Popup from 'terra-popup';
+import Button from 'terra-button';
+import Placeholder from 'terra-doc-template/lib/Placeholder';
 import classNames from 'classnames/bind';
 import styles from './PopupAttachmentBehavior.module.scss';
 
 const cx = classNames.bind(styles);
 
-class PopupAttachmentBehavior extends React.Component {
-  constructor(props) {
-    super(props);
+function PopupAttachmentBehavior() {
+  const buttonElement = useRef(null);
```

Comment:
You shouldn't need `null` here.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/927/commits/6f013aa3d8244f7caf366c84fa97c41acdefcbcb.